crates/server/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "server"
```

Comment:
not a great name for a crate. How about "steward-server"??

Reply:
It's short & sweet, and not being released. @rvolosatovs suggested that it to be called "server".

Reply:
I suggested `/crates/server` as placement, but, more importantly, I suggested following a similar approach as Drawbridge
Drawbridge server crate is called `drawbridge-server`, let's keep this consistent https://github.com/profianinc/drawbridge/blob/064afd4afada55b76f17398c62641c49a4b9f522/crates/server/Cargo.toml#L2

Reply:
Renamed.